demos/Multi_interface_demo/ReadMe.txt

```diff
@@ -0,0 +1,34 @@
+The FreeRTOS+TCP source code and example projects are currently provided in
```

Comment:
The readme should describe the new multi interface demo

Reply:
Indeed! I was looking into that only. I'll push the changes for it soon. Thanks.